src/features/CanvasEditor.tsx

```diff
@@ -1,29 +1,40 @@
 import { DndProvider } from 'react-dnd';
-import { createRef, useEffect, useState, useRef } from 'react';
-import { HTML5Backend } from 'react-dnd-html5-backend';
+import { useEffect, useState, useRef } from 'react';
 import { TouchBackend } from 'react-dnd-touch-backend';
-import { Button, Typography } from 'antd';
 import { ComponentManager } from './componentManager/ComponentManager';
 import { componentTypes } from './widgetsManager/component';
 import CanvasContainer from './canvasEditor/CanvasContainer';
 import { CustomDraggerLayer } from './canvasEditor/CustomDraggerLayer';
-// import { useEditorStore } from './stores/canvasStore';
-import { isEqual, debounce } from 'lodash';
+import { isEqual } from 'lodash';
```

Comment:
can we have direct import so we can avoid full library import
import isEqual from 'lodash/isEqual';